install/configure_administrative_access.md

```diff
@@ -111,18 +111,7 @@ CSM provide one or more layers of configuration in a process called "NCN persona
 
 See [Configure Non-Compute Nodes with CFS](../operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md).
 
-## 6. Upload Olympus BMC recovery firmware into TFTP server
```

Comment:
Need to update the table of contents for the removal of this section: https://github.com/Cray-HPE/docs-csm/blob/d0a8596ac958e3482c2fc8f114c072b312a59ad3/install/configure_administrative_access.md?plain=1#L17-L18